app/src/main/res/menu/action_bar.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
+<menu xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto">
+    <item android:id="@+id/logout"
+        android:icon="@drawable/internet"
```

Comment:
Daca nu ai inspiratie pentru iconite, te poate ajuta AndroidStudio:

1. File -> New -> Image Asset 
2. Selectezi Action Bar and Tab Icons la Icon type
3. Dai click pe iconita de la Clip Art si alegi tu ce iconita ti se pare potrivita (e.g. subdirectory seamana cu un logout)
4. Ii dai un nume sugestiv, e.g. ic_logout
5. Next -> Finish
6. Il folosesti aici in xml dupa aia